src/palette/qml/MuseScore/Palette/PalettePropertiesDialog.qml

```diff
@@ -111,6 +111,7 @@ StyledDialogView {
                         currentValue: modelData["value"]
                         measureUnitsSymbol: Boolean(modelData["measureUnit"]) ? modelData["measureUnit"] : ""
                         step: modelData["incrementStep"]
+                        minValue: step
```

Comment:
We should explicitly specify minValue/maxValue for Width/Height/Offset/Scale. @bkunda could you please help us with the Palette properties dialog as well?